facturacion_mexico/facturacion_fiscal/tests/test_bloque_e3_timbrado_uom.py

```
@@ -76,3 +80,63 @@ def test_mensaje_dice_antes_de_timbrar(self):
+
+	def test_legacy_extrae_codigo(self):
+		self.assertEqual(normalize_uom_to_sat_code("H87 Pieza"), "H87")
+		self.assertEqual(normalize_uom_to_sat_code("KGM Kilogramo"), "KGM")
+		self.assertEqual(normalize_uom_to_sat_code("E48 Unidad de Servicio"), "E48")
+
+	def test_codigo_puro_extrae_codigo(self):
+		self.assertEqual(normalize_uom_to_sat_code("H87"), "H87")
+		self.assertEqual(normalize_uom_to_sat_code("KGM"), "KGM")
+
+	def test_invalido_lanza_error(self):
+		with self.assertRaises(frappe.ValidationError):
```

Ruff (0.15.15)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)